### PR TITLE
Add related content to Flexible Content matrix field

### PR DESCRIPTION
**Pull request by mattandrews**
Re-do of https://github.com/biglotteryfund/craft-dev/pull/189 after the upgrade shenanigans

Looks like this:

image

I had some weirdness locally after creating this where certain fields got wiped out but I think this is down to my dev environment being a bit weird. That said I want to be quite cautious launching this so will be careful with backups etc.

Pairs with https://github.com/biglotteryfund/blf-alpha/pull/2383

### CODE REVIEW

File: modules/control-panel/Module.php [hunk 1]
```diff
@@ -9,6 +10,8 @@ class Module extends \yii\base\Module
 {
     public function init()
     {
+        // Define a custom alias named after the namespace
+        Craft::setAlias('@biglotteryfund', __DIR__);
```
Comment (mattandrews) [inline modules/control-panel/Module.php]: Came across this somewhere in Craft's github issues for newly-upgraded installs – was causing `./craft` commands to fail without it.

Comment (davidrapson) [reply modules/control-panel/Module.php]: Interesting, I spotted that error too. Is this module the right place for this line?

Comment (davidrapson) [reply modules/control-panel/Module.php]: (We don't need to answer this for this PR, merge away)

File: lib/ContentHelpers.php [hunk 2]
```diff
@@ -7,6 +7,18 @@
 
 class ContentHelpers
 {
+
+    private static function buildTrailImage($imageField)
```
Comment (davidrapson) [inline lib/ContentHelpers.php]: Good idea adding this as a helper

**Pull request merged**